src/components/Tooltip/index.js

```diff
@@ -141,47 +142,50 @@ class Tooltip extends PureComponent {
         TooltipSense.deactivate();
     }
 
+    /**
+     * Returns true if children is a focusable component.
+     *
+     * @returns {Boolean}
+     */
+    isFocusable() {
+        const name = this.props.children.type.displayName;
+        const isPressableText = name === 'Text' && Boolean(this.props.children.props.onPress);
+        return isPressableText || ['TouchableOpacity', 'Pressable', 'TouchableWithoutFeedback'].includes(name);
+    }
+
```

Comment:
We can't just simply check if it's a single child and decide to not wrap it with a view as I explained here https://github.com/Expensify/App/pull/16052#issuecomment-1472464715. So, if we have this component.

```
<Tooltip>
  <Avatar />
</Tooltip>
```

it will become like this

```
<Context.Provider onBlur onMouseEnter onMouseLeave etc..>
  <Avatar />
</Context.Provider>
```

and the callback does not work for context provider, so the tooltip won't show.

Reply:
Wait, so we still use the `absolute` props? Sorry if I misunderstood your intention. Btw, I will continue to look at this tomorrow as it's EOD for me.